tests/extract_status.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Extract generated status into a signac project for simplified inspection."""
+
+import argparse
+import os
+
+import generate_status_reference_data as gen
+import signac
+
+PROJECT_DIR = os.path.join(os.path.dirname(__file__), "./status_reference_data")
+
+
+def main(args):
+    if not os.path.exists(PROJECT_DIR):
+        os.makedirs(PROJECT_DIR)
+    elif args.force:
+        import shutil
+
+        shutil.rmtree(PROJECT_DIR)
+        os.makedirs(PROJECT_DIR)
+    else:
+        return
+
+    p = signac.init_project(name=gen.PROJECT_NAME, root=PROJECT_DIR)
```

Comment:
There are two projects and we're conflating the names. In the other file, `PROJECT_NAME` is used for the project holding state points with `a` and `b` (the FlowProject whose status is printed). In this file, `PROJECT_NAME` is being used for the project holding status options in the state point. I suggest creating two variables in the generation script, perhaps something like:
```python
STATUS_OPTIONS_PROJECT_NAME = "StatusOptionsData"
PROJECT_NAME = "StatusTest"
```
Then this file would use `gen.STATUS_OPTIONS_PROJECT_NAME` on this line.
```suggestion
    p = signac.init_project(name=gen.STATUS_OPTIONS_PROJECT_NAME, root=PROJECT_DIR)
```